docs/rules/html-attributes-casing.md

```diff
@@ -0,0 +1,35 @@
+# Define a style for the attributes casing in templates. (html-attributes-casing)
```

Comment:
@mysticatea there is bug: mysticatea/vue-eslint-parser#12 with parsing `argument` for `VDirectiveKey`